egs/material/s5/local/rnnlm/run_tdnn_lstm.sh

```diff
@@ -29,7 +29,7 @@
 embedding_dim=512
```

Comment:
@mahsa7823 , Are the results above up-to-date?

Reply:
I think so. Unless Hainan will have a new update after resolving a possible bug in rnnlm. @hainan-xv